l10n_hr_account_base/models/account_tax.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+class AccountTax(models.Model):
+    '''Inherit Account tax to add notes field to the taxes '''
+    _inherit = "account.tax"
+
+    l10n_hr_tax_notes = fields.Text(
+        string='Notes',
```

Comment:
staviti help i mozda placeholder da jasno znamo svrhu polja
```suggestion
        string='Tax Notes',
```

Reply:
fali ti prijevod